hmcts/pre-api
# Endpoint to check if ism file exists for use in edit flow

### JIRA ticket(s)

- S28-3144

### Change description

- Exposes an endpoint to check if the ism file exists in a given container
- This is used by the App_API_EditedVideoContainer flow for polling to see if an edited video has been uploaded

> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**

- Create an empty directory in the final storage account
- Call GET /media-service/blob/{container name} and observe a 404 response
- Upload a .ism file to the same container created before
- Call GET /media-service/blob/{container name} and observe a 204 response
